solr/core/src/java/org/apache/solr/cloud/overseer/NodeMutator.java

```diff
@@ -38,10 +38,6 @@
 
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-  public NodeMutator(ZkStateReader zkStateReader) {
-
-  }
-
```

Comment:
I wouldn't bother cleaning this up, just makes it trickier to merge.  Feel free to remove this in whatever patch you submit to the public solr JIRA